lesson27/test_search.py

```diff
@@ -0,0 +1,10 @@
+from tracking_page import TrackingPage
+
+def test_tracking_status(driver):
+    driver.get("https://tracking.novaposhta.ua/#/uk")
```

Comment:
there should be no driver in the test
this should be extracted to the method on the page
the test should reflect user behavior like open page, enter number, check result...